.travis.yml

```diff
@@ -20,7 +21,12 @@ script:
 - python3 motiontest.py --thermal-config-file smoketest/config.toml -c smoketest/test-config.yaml smoketest/clips/hedgehog.cptv
 - sudo mkdir /var/spool/cptv
 - sudo /home/travis/virtualenv/python3.6.7/bin/python3 piclassify.py --cptv smoketest/clips/hedgehog.cptv  --thermal-config-file smoketest/config.toml -c smoketest/test-config.yaml
-# - python3 classify.py all -c smoketest/test-config.yaml need model  
+- |
```

Comment:
I am not running these tests on travis, i'm not sure there is any point? but if we are happy with thests it may be worth putting the results for our current tracking somewhere, for easy comparison